<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analyse_tab</class>
 <widget class="QWidget" name="analyse_tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>987</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyse</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="analysis_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="title">
      <string>Analysis Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="3">
       <widget class="QGroupBox" name="algorithm_group">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Algorithm</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="result_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Show Result</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="start_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Start Analysis</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="algo_param_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Algorithm Parameters</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QGroupBox" name="run_group">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="title">
            <string>Current Run</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="chisquare_label">
              <property name="text">
               <string>Chisquare</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="chisquare_output">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="decimals">
               <number>8</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QProgressBar" name="progress_bar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="format">
               <string>%p%</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Simulation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0" colspan="3">
          <widget class="QPushButton" name="simulation_button">
           <property name="text">
            <string>Simulate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="2" colspan="3">
          <widget class="QGroupBox" name="simulation_wave_group">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Wavelength Range (nm)</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="min_label">
              <property name="text">
               <string>min</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="max_label">
              <property name="text">
               <string>max</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="min_input">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>350</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="max_input">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>1100</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="points_label">
              <property name="text">
               <string>Number of Points</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="point_input">
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>1500</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QTableWidget" name="analysis_table">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>140</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Layer Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Refractive Index Source</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Number of Term</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Parameters</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Delete</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QFrame" name="button_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="add_layer_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Layer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_all_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete Stack</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2">
       <widget class="QFrame" name="layer_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scroll_area_layers">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scroll_area_layers_contents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>159</width>
              <height>237</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="index_group">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Refractive Index Source</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="1">
             <widget class="QSpinBox" name="fb_spinbox">
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QRadioButton" name="fb_radio_button">
              <property name="text">
               <string>FB Model</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="fb_label">
              <property name="text">
               <string>Number of Peaks </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="mat_radio_button">
              <property name="text">
               <string>MAT Files</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QComboBox" name="mat_combobox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="graph_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Graphs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QTabWidget" name="graph_tab_holder">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fb_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>fb_spinbox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fb_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>fb_label</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fb_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>mat_combobox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mat_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>mat_combobox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mat_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>fb_spinbox</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mat_radio_button</sender>
   <signal>clicked()</signal>
   <receiver>fb_label</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
